From 001abcdca2026d0e1f51ca4e4e9d2cff052eadd7 Mon Sep 17 00:00:00 2001
From: Jan Cholasta <jcholast@redhat.com>
Date: Thu, 4 Aug 2016 14:14:15 +0200
Subject: [PATCH] vault: add missing salt option to vault_mod

The option was accidentally removed in commit
4b119e21a2f93ca16c5edf3d1058552b44feeaf8.

https://fedorahosted.org/freeipa/ticket/6154

Reviewed-By: Jan Cholasta <jcholast@redhat.com>
---
 ipaclient/plugins/vault.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipaclient/plugins/vault.py b/ipaclient/plugins/vault.py
index 73ad09b38316d55b466b7973dbeffefc1b7bb528..9026cbb0829a7557584df27a4262dfde640b4f28 100644
--- a/ipaclient/plugins/vault.py
+++ b/ipaclient/plugins/vault.py
@@ -413,7 +413,7 @@ class vault_mod(Local):
 
     def get_options(self):
         for option in self.api.Command.vault_mod_internal.options():
-            if option.name not in ('ipavaultsalt', 'version'):
+            if option.name != 'version':
                 yield option
         for option in super(vault_mod, self).get_options():
             yield option
-- 
2.7.4

